charts/agent/templates/_daemonset-logs-metrics-config.tpl

```diff
@@ -168,11 +168,11 @@ service:
         receivers: [filelog]
         processors: [memory_limiter, batch, resourcedetection/cloud, k8sattributes, attributes/observe_common, attributes/debug_source_pod_logs]
         exporters: [otlphttp/observe/base, debug/override]
-      hostmetrics:
+      metrics/hostmetrics:
```

Comment:
ah sorry, just saw this...
Sorry for the bug!
I uploaded a PR before seeing your PR...https://github.com/observeinc/helm-charts/pull/192

Reply:
ok - reverted commit